front/package-lock.json

```diff
@@ -4,7 +4,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "front",
```

Comment:
Do we want to get rid of this?

Reply:
This one is a bit stochastic it comes an goes